### ISSUE TITLE
Access Denied

### ISSUE THREAD

**Issue by jimmothy-beep** [DIGITALCRIMINAL/OnlyFans#332]
I have tried re-downloading the latest commit and going through the complete setup process again but I am still getting access denied when I attempt to connect

**Comment by edmundackah** [comment]
Same here

**Comment by t3chbr0** [comment]
Same problem here

**Comment by herefromrdt** [comment]
Not guaranteeing this is your problem, but every time I have this issue, it's because the User-Agent I'm using is outdated. Try making sure the User-Agent is up to date with the browser you logged in with.

**Comment by Alister-Zeus** [comment]
Had this issue with Python 3.7.2 changing to python 3.8.5 fixed it.

(Now to fix having multiple python version on PC and 3.8.5 not on path)

**Comment by bobsledtime** [comment]
Same issue. Python 3.8.5. Verified user agent and cookie details.

**Comment by turtlechase** [comment]
same here

**Comment by DatDirectorGuy** [comment]
Could someone explain how to verify user agent? ELI5 lol

**Comment by dylan-yue** [comment]
Python 3.8.2
Updated sess and user-agent

Still have the problem, saying "access denied".

(It worked several days ago, and the scraper is updated)

**Comment by bluebower** [comment]
Having the same issue on Python 3.8.0, and by updating my user agent in config.

**Comment by minayondaim3** [comment]
Having this issue as well

**Comment by codykeler** [comment]
Deleting cookies and logging in again in the browser fixed the issue for me. Apparently I had an outdated cookie or whatever.
